### ISSUE TITLE
Does not support "firebase-admin": "^9.3.0"

### ISSUE THREAD

**Issue by breshman** [tfarras/nestjs-firebase-admin#2]
**Please can you update as it does not support "firebase-admin": "^9.3.0"**

   ~~~~~~~

[9:21:42] Starting compilation in watch mode...

node_modules/@google-cloud/firestore/types/firestore.d.ts:25:1 - error TS6200: Definitions of the following identifiers conflict with those in another file: DocumentData, UpdateData, Firestore, GeoPoint, Transaction, WriteBatch, WriteResult, DocumentReference, DocumentSnapshot, QueryDocumentSnapshot, OrderByDirection, WhereFilterOp, Query, QuerySnapshot, DocumentChangeType, CollectionReference, FieldValue, FieldPath, Timestamp, v1beta1, v1, OK, CANCELLED, UNKNOWN, INVALID_ARGUMENT, DEADLINE_EXCEEDED, NOT_FOUND, ALREADY_EXISTS, PERMISSION_DENIED, RESOURCE_EXHAUSTED, FAILED_PRECONDITION, ABORTED, OUT_OF_RANGE, UNIMPLEMENTED, INTERNAL, UNAVAILABLE, DATA_LOSS, UNAUTHENTICATED, FirebaseFirestore

25 declare namespace FirebaseFirestore {


  node_modules/@tfarras/nestjs-firebase-admin/node_modules/@google-cloud/firestore/types/firestore.d.ts:23:1
    23 declare namespace FirebaseFirestore {

    Conflicts are in this file.

node_modules/@google-cloud/firestore/types/firestore.d.ts:168:5 - error TS2374: Duplicate string index signature.

168     [key: string]: any; // Accept other properties, such as GRPC settings.


node_modules/@tfarras/nestjs-firebase-admin/node_modules/@google-cloud/firestore/types/firestore.d.ts:23:1 - error TS6200: Definitions of the following identifiers conflict with those in another file: DocumentData, UpdateData, Firestore, GeoPoint, Transaction, WriteBatch, WriteResult, DocumentReference, DocumentSnapshot, QueryDocumentSnapshot, OrderByDirection, WhereFilterOp, Query, QuerySnapshot, DocumentChangeType, CollectionReference, FieldValue, FieldPath, Timestamp, v1beta1, v1, OK, CANCELLED, UNKNOWN, INVALID_ARGUMENT, DEADLINE_EXCEEDED, NOT_FOUND, ALREADY_EXISTS, PERMISSION_DENIED, RESOURCE_EXHAUSTED, FAILED_PRECONDITION, ABORTED, OUT_OF_RANGE, UNIMPLEMENTED, 
INTERNAL, UNAVAILABLE, DATA_LOSS, UNAUTHENTICATED, FirebaseFirestore

23 declare namespace FirebaseFirestore {
  node_modules/@google-cloud/firestore/types/firestore.d.ts:25:1
    25 declare namespace FirebaseFirestore {

    Conflicts are in this file.

[9:21:49] Found 3 errors. Watching for file changes.
   ~~~~~~~

**Comment by hkdobrev** [comment]
Any updates on this? It seems to not support major version 9 of `firebase-admin`. And they are not backporting neither fixes nor features on the `8.x` release line.

**Comment by breshman** [comment]
Crea una carpeta de esta forma en tu proyecto y luego copia y pega todos los archivos. Revisa la imagen adjunta. 

![image](https://user-images.githubusercontent.com/34691543/100180835-aac9e280-2e9e-11eb-9979-e045a3f3b1ba.png)

**Comment by hkdobrev** [comment]
I don't think this is a sustainable workaround. It's about addressing breaking changes and updating the version constraint so newer versions could be used with patches.

**Comment by breshman** [comment]
puede ser una solución hasta que actualicen, hace un mes mande las actualizaciones pero hasta ahora no lo unen (merge).

**Comment by chrisbra-riftpay** [comment]
This is working for 9.4.1

**Comment by tfarras** [comment]
Fixed with #4
